musicname/ze_ffvii_malgo_reactor_css5.cfg

```
@@ -0,0 +1,35 @@
+"music"
+{
+    "dj_fresh_the_power.mp3"                    "DJ Fresh & Dizzee Rascal - The Power",
+    "ffvii_chopperhead1.mp3"                    "Two Steps From Hell - Chopperhead"
+    "ffvii_ifrit1.mp3"                          "FFXV OST - Ifrit Boss Theme"
+    "ffviii_ac_version_those_who_fight.mp3"     "FFVII: Advent Children OST - Those Who Fight"
+    "pendulum_9_000_miles.mp3"                  "Pendulum - 9000 Miles"
+    "pendulum_bloodsugar.mp3"                   "Pendulum - Blood Sugar"
+    "pendulum_granite.mp3"                      "Pendulum - Granite"
+    "pendulum_napalm.mp3"                       "Pendulum, Joey Valence & Brae - Napalm"
+    "pendulum_propane_nightmares.mp3"           "Pendulum - Propane Nightmares"
+    "pendulum_salt_in_the_wounds.mp3"           "Pendulum - Salt In The Wounds"
+    "pendulum_self_vs_self.mp3"                 "Pendulum - Self vs Self (ft. In Flames)"
+    "pendulum_showdown.mp3"                     "Pendulum - Showdown"
+    "pendulum_slam.mp3"                         "Pendulum - Slam"
+    "pendulum_tarantula.mp3"                    "Pendulum - Tarantula"
+    "pendulum_tempest.mp3"                      "Pendulum - The Tempest (Live at Brixton Academy)"
+    "pendulum_the_fountain.mp3"                 "Pendulum - The Fountain"
+    "pendulum_the_island_to_glory.mp3"          "Pendulum - The Island, Two Steps From Hell - To Glory"
+    "pendulum_watercolour.mp3"                  "Pendulum - Watercolour"
+    "pendulum_witchcraft.mp3"                   "Pendulum - Witchcraft"
+    "prey_sub_focus_illuminate.mp3"             "Sub Focus & Wilkinson - Illuminate"
+    "rmzs_boss.mp3"                             "Muzzy - Play (ft. UK:ID)"
+    "rmzs_end.mp3"                              "MUZZ - Endgame"
+    "rmzs_first.mp3"                            "MUZZ - Junction Seven"
+    "rmzs_second.mp3"                           "MUZZ - Play (ft. UK:ID), MUZZ - Back To You"
```

`Back to You` was made by both Muzzy and Voicians.
```suggestion
    "rmzs_second.mp3"                           "MUZZ - Play (ft. UK:ID), Muzzy & Voicians - Back To You"
```